src/lib/get-costume-url.js

```diff
@@ -26,10 +26,12 @@ const getCostumeUrl = (function () {
                 svgRenderer.loadString(svgString);
                 const svgText = svgRenderer.toString(true /* shouldInjectFonts */);
                 cachedUrl = `data:image/svg+xml;utf8,${encodeURIComponent(svgText)}`;
+            } else {
+                cachedUrl = vm.runtime.storage.get(assetId).encodeDataURI();
             }
+        } else {
             cachedUrl = vm.runtime.storage.get(assetId).encodeDataURI();
         }
-        cachedUrl = vm.runtime.storage.get(assetId).encodeDataURI();
```

Comment:
Maybe
if (!cachedUrl) cachedUrl = ...
would be easier to reason about than the two elses, but I don't feel strongly about it

Reply:
I think I'm going to leave it as is.